store/src/main/java/com/dropbox/android/external/store4/StoreResponse.kt

```diff
@@ -41,16 +41,25 @@ sealed class StoreResponse<out T> {
     /**
      * Error dispatched by a pipeline
      */
-    data class Error<T>(val error: Throwable, override val origin: ResponseOrigin) :
-        StoreResponse<T>()
+    sealed class Error<T> : StoreResponse<T>() {
+        data class Exception<T>(
```

Comment:
my immediate reaction to this was that we should not name it Exception.
but thinking more, i'm not sure what is better, i might be over-reacting but this might significantly irritate people.

Reply:
I don't have a better suggestion either 🤔

Reply:
i think this is OK for now then.